src/h3/h3core.pyx

```diff
@@ -1,8 +1,13 @@
 cimport h3lib
-from .h3utils cimport check_addr, check_edge, check_res, create_mv, create_ptr
-
-from cpython cimport bool
-from libc.stdint cimport int64_t
+from h3lib cimport bool, int64_t, H3int
```

Comment:
Should `bool` and `int64_t` still be imported from `cpython` and `libc.stdint`?

Reply:
These types seemed out of place in `h3core.pyx`, so I wanted to put all the type declarations in one place. `h3lib.pxd` seemed like the best candidate.

Reply:
I don't think it should be an issue to import types in `pyx` files, with this it feels like we're doing `stdint.pxd`->`h3lib.pxd`->`h3core.pyx` instead of `stdint.pxd`->`h3core.pyx`

Reply:
That's exactly what we're doing ;)

I'm thinking of it more like `(whatever random sources)` -> `h3lib.pxd` -> `h3core.pyx`. All types are then defined in `h3lib.pxd`.

I like that better than having types come from three different sources (`cpython`, `stdint`, `h3lib`) -> `h3core.pyx`.

If other `.pyx` files need to use the same types, they can all import from `h3lib`, without having to remember they're hidden away in `cpython` or `stdint`.